tests/agent/filter/registry_test.py

```diff
@@ -0,0 +1,31 @@
+from opentelemetry.trace import Span, NonRecordingSpan
```

Comment:
Let's name the python test files using the same naming convention test_blah_1.py, test_blah_2.py that are used for the other modules.

Reply:
Updated to `test_registry`, I don't think the `_1` should be necessary here since this is more of an individual module test, unlike a library specific set of test files?